### PR TITLE
[IMPAC-126] Remove bootstrap spacer divs from impac-angular

### PR DESCRIPTION
**Pull request by claire00**
[MNOE-101] & [IMPAC-126]
@alexnoox @xaun

### CODE REVIEW

File: src/components/dashboard/dashboard.tmpl.html [hunk 1]
```diff
@@ -151,6 +147,7 @@
 
   <!-- First Time Dashboard Creation -->
   <div class="row text-center" ng-show='showChooseDhbMsg()'>
+    <img class="bg" ng-src="{{dhbBackground}}">
```
Comment (xaun) [inline src/components/dashboard/dashboard.tmpl.html]: @alexnoox @claire00 where is `dhbBackground` defined?

File: src/stylesheets/variables.less [hunk 2]
```diff
@@ -59,6 +59,12 @@
 @impac-dashboard-source-color:                  @impac-dashboard-title-label-color;
 @impac-dashboard-buttons-border-radius:         4px;
 
+//Info-text
+@impac-dashboard-info-text-color:               white;
+@impac-dashboard-info-text-background-color:    rgb(23,38,45);
```
Comment (xaun) [inline src/stylesheets/variables.less]: @alexnoox @claire00 maybe you could use `rgb(23,38,45, 0.7)` so then the opacity can be customised with a variable, and there is no need to use the `opacity: 0.7;` rule in the `dashboard.less` file.

File: src/components/dashboard/dashboard.tmpl.html (hunk 1 above)
Comment (claire00) [inline src/components/dashboard/dashboard.tmpl.html]: @xaun @alexnoox My fault, I will commit again.

**Comment by xaun** [comment]
Good work 👍 I will merge this into 1.4 later this week

**Comment by alexnoox** [comment]
LGTM. Well done!

**Comment by xaun** [comment]
@claire00 closing PR as merged in with https://github.com/maestrano/impac-angular/pull/244

**Pull request closed without merging**